conference.js

```diff
@@ -638,8 +657,8 @@ export default {
                     // Instead of disabling the button we want to show button
                     // muted, so that the user can have the opportunity to add
                     // the video later on, even if joined without it.
-                    this.videoMuted = true;
-                    APP.UI.setVideoMuted(this.getMyUserId(), this.videoMuted);
+                    APP.UI.setVideoMuted(
+                        this.getMyUserId(), this.videoMuted = true);
```

Comment:
Do you prefer the assignment happen in the function invocation? What advantage do you see over leaving the assignment on a separate line?